src/SSW.SophieBot/components/SSWSophieBot.HttpClientComponents.PersonQuery/Actions/GetGroupedEmployeesAction.cs

```diff
@@ -75,7 +75,9 @@ public override async Task<DialogTurnResult> BeginDialogAsync(DialogContext dc,
                         .ToList();
 
                     Func<GetEmployeeModel, string, bool> countFunc = (e, s) 
-                        => e.Skills.Any(es => es.ExperienceLevel == "Advanced" && es.Technology.Equals(s, StringComparison.OrdinalIgnoreCase));
+                        => e.Skills.Any(es => es.ExperienceLevel == "Advanced" 
```

Comment:
What problem does it solve? Didn't see any checklist item related to this on #20

Reply:
It solves a potential bug when technology field is null